packages/dapp/src/store/reducer.ts

```diff
@@ -31,6 +32,7 @@ export interface State {
   switchEncryptionKey: Function;
   dappConfig: string;
   switchDappConfig: Function;
+  node?: IPFS;
```

Comment:
It is better to name this property `ipfsNode` because in the future in the global state may appear another nodes kind.